frontend/src/app/features/admin/admin.module.ts

```diff
@@ -15,12 +16,13 @@ const DECLARATIONS = [
 
 const IMPORTS = [
   AdminRoutingModule,
-  SharedModule
+  SharedModule,
+  TranslocoModule
```

Comment:
Ezt nézd meg, hogy lehet-e singleton. Ha igen akkor core-ba, ha nem akkor shared-be tedd kérlek.